sensor_msgs/Image image
---
bool success
string message
YOLOResult result